### PR TITLE
grunt-minifyify

### PR DESCRIPTION
**Pull request by ajablon**

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    }
+  });
+
+  grunt.loadNpmTasks('grunt-contrib-clean');
+  grunt.loadNpmTasks('grunt-contrib-nodeunit');
+  grunt.loadNpmTasks('grunt-browserify');
+  grunt.loadTasks("tasks");
+
+  grunt.registerTask("test", ["clean", "browserify", "minifyify:builtin", "minifyify:separate", "nodeunit"]);
+};
```
Comment (m-vdb) [inline Gruntfile.js:45]: I don't understand why everything is hardcoded here (foo / bar). The GruntFile is only useful for testing, is that it ?

File: tasks/minifyify.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+                }
+              }
+            )
+          );
+        } else {
+          var writeStream = fs.createWriteStream(dest.buildFile);
+
+          readStream
+          .on('open', function(){
+          readStream.pipe(minifyify(options)).pipe(writeStream);
```
Comment (m-vdb) [inline tasks/minifyify.js]: indentation

File: tasks/minifyify.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+          })
+          .on('error', function(err){
+          grunt.log.error(err);
+          done(false);
```
Comment (m-vdb) [inline tasks/minifyify.js]: indentation

File: tasks/minifyify.js [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+          });
+
+          writeStream.on('finish', function(){
+          grunt.log.ok('Minifyified ' + dest.buildFile);
+          next();
```
Comment (m-vdb) [inline tasks/minifyify.js]: indentation

File: Gruntfile.js (hunk 1 above)
Comment (ajablon) [inline Gruntfile.js:45]: that's it! (cf. i.e. https://github.com/gruntjs/grunt-contrib-uglify/blob/master/Gruntfile.js)

**Pull request merged**